src/components/ui/Panel.jsx

```diff
@@ -211,27 +207,17 @@ class Panel extends React.Component {
             onTransitionEnd={() => this.updateMobileMapUI()}
             {...(isMobile && resizable && this.getEventHandlers())}
           >
-            <Flex
-              justifyContent="space-between"
-              className={classnames(
-                'panel-header',
-                { 'panel-resizeHandle': resizable && isMobile }
-              )}
-              onClick={() => isMobile && this.handleHeaderClick()}
-            >
-              {resizable && isMobile && size === 'minimized' && minimizedTitle
-                ? <span className="minimizedTitle">{minimizedTitle}</span>
-                : title}
-              {close &&
-              <Flex
-                justifyContent="center"
-                className="panel-close"
-                title={_('Close')}
-                onClick={e => {e.stopPropagation(); close(e);}}
+            {isMobile &&
+              <div
+                className="panel-drawer"
+                onClick={() => this.handleHeaderClick()}
               >
-                <i className="icon-x" />
-              </Flex>}
-            </Flex>
+                {resizable && isMobile && <div className="panel-handle"/>}
```

Comment:
Related to my comment about the grey handle being a pseudo-element, this could simply be replaced by an optional classname on `.panel-drawer`.